Shared/Samples/Control annotation sublayer visibility/ControlAnnotationSublayerVisibilityView.Model.swift

```diff
@@ -0,0 +1,112 @@
+    @Observable
+    class Model {
+        /// A map from the mobile map package.
+        private(set) var map = Map()
+        
+        /// The mobile map package.
+        @ObservationIgnored private var mobileMapPackage: MobileMapPackage!
+        
+        /// The annotation layer in the map.
+        @ObservationIgnored private var annotationLayer: AnnotationLayer?
+        
+        /// The closed annotation sublayer.
+        @ObservationIgnored private var closedSublayer: AnnotationSublayer?
+        
+        /// The open annotation sublayer.
+        @ObservationIgnored private var openSublayer: AnnotationSublayer?
+        
+        /// A Boolean value indicating whether to show the closed annotation sublayer.
+        var showsClosedSublayer = true
+        
+        /// A Boolean value indicating whether to show the open annotation sublayer.
+        var showsOpenSublayer = true
+        
+        /// A Boolean value indicating whether the open annotation sublayer is visible at the current extent.
+        var visibleAtCurrentExtent = false
+        
+        /// The current scale of the map.
+        @ObservationIgnored var currentScale: Double = .zero {
+            didSet {
+                currentScaleText = "1:\(currentScale.formatted(.decimal))"
+                guard let openSublayer else { return }
+                visibleAtCurrentExtent = openSublayer.isVisible(atScale: currentScale)
+            }
+        }
+        
+        /// The map's current scale value in text.
+        var currentScaleText = ""
+        
+        /// The min and max scale of the open annotation layer in text.
+        var minMaxScaleText = ""
+        
+        /// Loads a local mobile map package.
+        func loadMobileMapPackage() async throws {
+            mobileMapPackage = MobileMapPackage(fileURL: .gasDevicePackage)
+            try await mobileMapPackage.load()
+            guard let map = mobileMapPackage.maps.first else { return }
+            self.map = map
+        }
+        
+        /// Sets the map's annotation layer and sublayers.
+        func setAnnotationSublayers() {
+            annotationLayer = map.operationalLayers.first(where: { $0 is AnnotationLayer }) as? AnnotationLayer
+            closedSublayer = annotationLayer?.subLayerContents[0] as? AnnotationSublayer
+            openSublayer = annotationLayer?.subLayerContents[1] as? AnnotationSublayer
+            setScaleText()
+        }
+        
+        /// Sets the visibility of the closed annotation sublayer.
+        /// - Parameter visibility: The visibility of the sublayer.
+        func setClosedSublayerVisibility(_ visibility: Bool) {
+            closedSublayer?.isVisible = visibility
+        }
+        
+        /// Sets the visibility of the open annotation sublayer.
+        /// - Parameter visibility: The visibility of the sublayer.
+        func setOpenSublayerVisibility(_ visibility: Bool) {
+            openSublayer?.isVisible = visibility
+        }
```

Comment:
A better name for the parameter may be `isVisible`, rather than `visibility`, as [visibility](https://developer.apple.com/documentation/swiftui/visibility) sounds like an enum. Update the docstring if the change is applied.